lib/mix/tasks/nerves.loaddefconfig.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Mix.Tasks.Nerves.Loaddefconfig do
+  @shortdoc "Load nerves_defconfig into the Nerves artifact"
+
+  @moduledoc """
+  Load the nerves_defconfig configuration into the Nerves artifact.
+  This is the opposite of `make savedefconfig`.
```

Comment:
```suggestion
  @shortdoc "Load nerves_defconfig into the Nerves artifact"

  @moduledoc """
  Load the nerves_defconfig configuration into the Nerves artifact.
  This is the opposite of `make savedefconfig`.
```

I don't think this is quite right. This is only used during build and development but the phrasing of this doc makes it sound like I can load into an already compiled artifact which is not correct.

Maybe mirroring as `nerves.create_build` would be better? Or just changing doc to reflect this loads configuration to prepare for artifact build?